documentation/userguide/docs/metadata_forms.md

```diff
@@ -0,0 +1,116 @@
+# **Metadata Forms**
+
+**Introduction**
+
+Metadata forms allow users to add structured contextual information to various entities in the data.all platform. By creating and attaching metadata forms, user can standardize and enrich metadata in a customizable way.
+
+Metadata forms serve several key purposes:
+
+- Improve data discovery by enabling more consistent, complete, and meaningful metadata
+- Capture domain-specific or organizational metadata standards
+- Streamline metadata management workflows
+- Search among all entities in data.all based on attached metadata
+
+**Metadata Forms visibility**
+
+Visibility setting defines who can view and attach a metadata form to an entity.
+
+- Global visibility means the metadata form is visible and attachable to any entity by all users across the platform
+- Organization/Environment-Wide visibility limits the form to a specific organization/environment - it can only be seen by members of this organization/environment and attached to entities in that organization/environment
+- Team-Only visibility restricts the form to just members of a specific team, but does not restrict to which entities it can be attached to
```

Comment:
So it could be attached to datasets in different environments if the team is the same?

Reply:
Yes

Reply:
Maybe we should clarify that, to avoid questions